### ISSUE TITLE
Spaces inserted after itemization stop remainder of file from being processed correctly

### ISSUE THREAD

**Issue by krcools** [JuliaDocs/Documenter.jl#143]
I am on Julia 0.4.6:

```
julia> versioninfo()
Julia Version 0.4.6
Commit 2e358ce (2016-06-19 17:16 UTC)
Platform Info:
  System: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Xeon(R) CPU E5-1620 v3 @ 3.50GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
```

and `Documenter`:

```
julia> Pkg.status()
16 required packages:
 - Documenter                    0.1.6+             master
```

The following markdown is correctly processed by `Documenter`:

```
# Title of the Documentation

First intro line.

This is a first item.

``@docs
numfunctions
coordtype
``
```

Inserting an itemisation, however:

```
# Title of the Documentation


First intro line.

- This is a first item.


``@docs
numfunctions
coordtype
``
```

causes lines following the itemization to be prepended by a space, stopping the rest of the file from being processed correctly. The result is:

```

<a id='Title-of-the-Documentation-1'></a>

# Title of the Documentation


First intro line.


  * This is a first item.
 
 
 ``@docs
 numfunctions
 coordtype
 ``
```

**Comment by krcools** [comment]
Changing from CRLF to CR seems to solve the problem. Duplicate of #115 probably. Apologies for the noise.

**Comment by MichaelHatherly** [comment]
> Duplicate of #115 probably.

Possibly, though without any feedback on that one I can't say for certain.

---

This may be an upstream bug since Documenter is just using the built in markdown parser.

What happens if you do the following:

    julia> md = Markdown.parse(readall("file.md"));

    julia> println(Markdown.plain(md))
    # Title of the Documentation

    First intro line.

      * This is a first item.

    ```@docs
    numfunctions
    coordtype
    ```


where `file.md` contains

    # Title of the Documentation

    First intro line.

    - This is a first item.


    ```@docs
    numfunctions
    coordtype
    ```

Also, if it's possible for you to check with Julia 0.5 as well that would be great, perhaps this has already been fixed and just not backported.

**Comment by MichaelHatherly** [comment]
Using `unix2dos` to convert the line endings and testing against `0.4` and `0.5` only shows the bug on `0.4`, so I suppose I may have inadvertently fixed this in https://github.com/JuliaLang/julia/pull/17359. 

Not sure whether we'll be doing another `0.4.*` release at some point once the dust has settled from `0.5` or not. If so, then I could try extracting just the bugfix part of that PR. Otherwise this isn't really actionable in Documenter. @krcools, if it's also fixed for you on `0.5` then I'll close this issue since there's not much we can really do for the moment other than changing line endings.

**Comment by MichaelHatherly** [comment]
Upstream issue, fixed in Julia `0.5`.
